test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -336,6 +336,35 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			}
 		})
 
+		It("should validate aks-engine-provided sysctl configuration", func() {
```

Comment:
This is rather manual, but it's proof that our new sysctldConfig interface is doing exactly what we think it's doing.